.github/workflows/check-image-vulnerabilities.yml

```diff
@@ -49,44 +51,62 @@ jobs:
-          endpoint: ${{ vars.ACS_DOGFOODING_CENTRAL_URL }}
+          image: rhacs-eng/${{ matrix.image }}
+          version: ${{ inputs.version || github.ref_name }}
+          wait-limit: "7200"
+          summary-prefix: "Upstream:"
+          quay-bearer-token: ${{ secrets.QUAY_RHACS_ENG_BEARER_TOKEN }}
+          central-url: ${{ vars.ACS_DOGFOODING_CENTRAL_URL }}
 
-      - name: Install roxctl
-        uses: stackrox/roxctl-installer-action@v1
+      - name: Upload scan result artifact
+        if: always()
+        uses: actions/upload-artifact@v4
         with:
-          central-endpoint: ${{ vars.ACS_DOGFOODING_CENTRAL_URL }}
-          central-token: ${{ env.ROX_API_TOKEN }}
+          name: scan-result-quay.io_rhacs-eng_${{ matrix.image }}_${{ inputs.version || github.ref_name }}.json
+          path: scan-result.json
+          retention-days: 30
 
-      - name: Scan images for vulnerabilities
-        run: |
-          {
-            echo "### ${{ matrix.image }}:${{ inputs.version }}"
-            echo "<details><summary>Click to expand</summary>"
-            echo ""
-            # We need a monospaced font for the table layout, and `terraform` has
-            # nicer color highlighting than the default language agnostic code block.
-            echo '```terraform'
-            roxctl image scan --output=table --force \
-              --image="quay.io/rhacs-eng/${{ matrix.image }}:${{ inputs.version }}" \
-              --severity="MODERATE,IMPORTANT,CRITICAL" \
-              --headers="COMPONENT,VERSION,CVE,SEVERITY,FIXED_VERSION,LINK" \
-              --row-jsonpath-expressions="{result.vulnerabilities.#.componentName,result.vulnerabilities.#.componentVersion,result.vulnerabilities.#.cveId,result.vulnerabilities.#.cveSeverity,result.vulnerabilities.#.componentFixedVersion,result.vulnerabilities.#.cveInfo}"
-            echo '```'
-            echo "</details>"
-          } >> "$GITHUB_STEP_SUMMARY"
+  check-image-vulnerabilities-konflux:
+    name: "Konflux: ${{ matrix.image }}:${{ inputs.version || github.ref_name }}"
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed for stackrox/central-login to create the JWT token.
+      id-token: write
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          [
+            "release-central-db",
+            "release-collector",
+            "release-main",
+            "release-roxctl",
+            "release-scanner",
+            "release-scanner-db",
+            "release-scanner-db-slim",
+            "release-scanner-slim",
+            "release-scanner-v4",
+            "release-scanner-v4-db",
+            "release-operator",
+          ]
+    steps:
+      - name: Check image vulnerabilities
+        uses: stackrox/actions/release/scan-image-vulnerabilities@tm/ROX-image-vuln-check
+        with:
+          image: rhacs-eng/${{ matrix.image }}
+          version: ${{ inputs.version || github.ref_name }}
+          wait-limit: "11400"
```

Comment:
As compared to the old implementation, `wait-limit` is added to pick up images from tagged builds.
Do you think we should have auto-triggering for tagged builds given Konflux instability?

If this is manually triggered run, it would be nice to not wait for images but rather let the step fail if its image is absent. Is there a way to conditionally set zero waiting time in this case?